slidedeck/src/slides/03-react-state/06-lab-2.pug

```diff
@@ -0,0 +1,4 @@
+section
```

Comment:
Since this lab works with state, I thought it might be good to move it until after we go in depth about state. It _might_ be too much lecturing and not enough code though?

Reply:
I think it'll be fine this way